The port runs autoreconf; fix in both places in case that changes in the future.

--- acinclude.m4.orig	2020-06-24 00:16:48.000000000 -0700
+++ acinclude.m4	2020-06-24 00:16:56.000000000 -0700
@@ -38,6 +38,7 @@
 [AC_TRY_LINK(
 changequote(<<, >>)dnl
 <<#include <time.h>
+#include <stdlib.h>
 #if !HAVE_DECL_DAYLIGHT
 extern int daylight;
 #endif>>,
--- configure.orig	2020-06-24 00:14:41.000000000 -0700
+++ configure	2020-06-24 00:14:50.000000000 -0700
@@ -13685,6 +13685,7 @@
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <time.h>
+#include <stdlib.h>
 #if !HAVE_DECL_DAYLIGHT
 extern int daylight;
 #endif
